config/environments/development.rb

```diff
@@ -33,8 +33,20 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # Don"t care if the mailer can"t send.
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  host = ENV["host"]
+  config.action_mailer.default_url_options = { host: host, protocol: ENV["protocol"] }
+  ActionMailer::Base.smtp_settings = {
+    :address    => "smtp.gmail.com",
```

Comment:
```suggestion
    address: "smtp.gmail.com",
```Các line bên dưới tự xử lý nhé